Gems/MotionMatching/Code/Source/FeatureMatrixTransformer.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/std/containers/span.h>
+#include <Allocators.h>
+#include <FeatureMatrix.h>
+
+namespace EMotionFX::MotionMatching
+{
+    class FeatureMatrixTransformer
```

Comment:
nit: An API comment with what this is and how it's used would be great

Reply:
Added